src/middleware/AuthMiddleware.ts

```diff
@@ -1,36 +1,44 @@
 import { SessionKey } from 'ch-node-session-handler/lib/session/keys/SessionKey';
 import { SignInInfoKeys } from 'ch-node-session-handler/lib/session/keys/SignInInfoKeys';
 import { Session } from 'ch-node-session-handler/lib/session/model/Session';
-import { ISignInInfo } from 'ch-node-session-handler/lib/session/model/SessionInterfaces';
+import { ISignInInfo, IUserProfile } from 'ch-node-session-handler/lib/session/model/SessionInterfaces';
 import { NextFunction, Request, RequestHandler, Response } from 'express';
 import { injectable } from 'inversify';
 import { BaseMiddleware } from 'inversify-express-utils';
+import { loggerInstance } from './Logger';
 
 import { getEnvOrDefault } from 'app/utils/EnvironmentUtils';
 import { PENALTY_DETAILS_PAGE_URI } from 'app/utils/Paths';
 
 function buildReturnUri(req: Request): string {
-    return new URL(PENALTY_DETAILS_PAGE_URI, `${req.protocol}://${req.headers.host}`).href
+    return new URL(PENALTY_DETAILS_PAGE_URI, `${req.protocol}://${req.headers.host}`).href;
 }
 
 @injectable()
 export class AuthMiddleware extends BaseMiddleware {
 
     public handler: RequestHandler = (req: Request, res: Response, next: NextFunction): void => {
-        req.session.ifNothing(() => console.log(`${req.url}: Session object is missing!`));
+        req.session
+            .ifNothing(() => loggerInstance().info(`${AuthMiddleware.name} - handler: Session object is missing!`));
```

Comment:
I doubt this should be logged at info level. There will be plenty instances where session won't be there and seeing `Session object is missing!` in logs does not give us much. More debug level for me.